components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -92,6 +93,10 @@ open class GleanInternalAPI internal constructor () {
         initializeCoreMetrics(applicationContext)
 
         ProcessLifecycleOwner.get().lifecycle.addObserver(gleanLifecycleObserver)
+
+        if (BuildConfig.DEBUG) {
```

Comment:
We don't want this to only work in debug builds. We'll need this for QAing final builds as well.

Reply:
Ok.  No problem.